Tools/Scripts/webkitruby/jsc-stress-test-writer-default.rb

```diff
@@ -252,7 +252,7 @@ def reproScriptCommand
     end
 
     def statusCommand(status)
-        "echo #{$runUniqueId} $exitCode #{status} > #{statusFile}"
+        "echo #{@index} #{$runUniqueId} $exitCode #{status} >> #{statusFile}"
```

Comment:
Hmm, I wonder if this is going to result in an ever-growing `statusFile` for CI systems. One way to work around this would be to include the `$runUniqueId` in the filename and remove it when done (may leave a stale file behind though). Alternatively, remove the `statusFile` before starting?

Reply:
Now it will clear the temp folder every time when it finished processing the results.